plants/whatsnew.md

```diff
@@ -1,9 +1,9 @@
 
 - New genomes
 
-    - [White Fonio](/Digitaria_exilis) (_Digitaria exilis_) : assembly from ENA [GCA_902859565.1](https://www.ebi.ac.uk/ena/browser/view/GCA_902859565.1) and annotation from [KAUST](https://www.kaust.edu.sa/en).
-    - [Cowpea](/Vigna_unguiculata) (_Vigna unguiculata_) : assembly from ENA [GCA_004118075.1](https://www.ebi.ac.uk/ena/browser/view/GCA_004118075.1) and annotation from [JGI](https://phytozome-next.jgi.doe.gov/cowpeapan/info/Vunguiculata_v1_2). 
-    - [Barnyard Grass](/Echinochloa_crusgalli) (_Echinochloa crusgalli_) : assembly and from NCBI [GCA_020466025.1](https://www.ncbi.nlm.nih.gov/assembly/GCA_020466025.1) and annotation from [ZJU](http://ibi.zju.edu.cn/RiceWeedomes/Echinochloa/).
+    - [Ryegrass](/Lolium_perenne) (_Lolium perenne_) : assembly from ENA [GCA_019359855.1](https://www.ncbi.nlm.nih.gov/assembly/GCA_019359855.1) and annotation from [Cyverse](https://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/Copetti_Kyuss_assembly_annotation_March_2021).
+    - [Brown Mustard](/Brassica_juncea) (_Brassica juncea_) : assembly from ENA [GCA_018703725.1](https://www.ncbi.nlm.nih.gov/assembly/GCA_018703725.1) and annotation from [oilseedhunan](http://www.oilseedhunan.net/download.html).
+    - [Cork Oak](/) (_Echinochloa crusgalli_) :  assembly and annotation from NCBI [GCA_002906115.4](https://www.ncbi.nlm.nih.gov/assembly/GCA_002906115.4).
```

Comment:
@gnaamati  - can you please check if this line is correct and has the correct URL set?

Reply:
Hi @msmanoj, which line are you referring to?

Reply:
I've updated the file to remove one species

Reply:
It looks like `Cork Oak` is missing the url after these changes. Is that expected?